docs/README.md

```diff
@@ -35,6 +35,7 @@ The props below support [`responsive-values`](https://github.com/vtex-apps/respo
 | `preventVerticalStretch`   | `Boolean`             | Prevents the row from stretching vertically to fill its parent height with `items-stretch` token.             | `false`       |
 | `horizontalAlign`          | `left`&#124;`right`&#124;`center` | Controls horizontal alignment for the items inside the `flex-layout.row`.                                              | `left`        |
 | `colSizing`                | `equal`&#124;`auto`       | Controls the width of the columns inside the `flex-layout.row`.                                                        | `equal`       |
+| `colJustify`                | `between`&#124;`around`       | Controls the automatic space between columns and borders of the `flex-layout.row`.                                                        | `between`       |
```

Comment:
Since there are 2 specified values for the prop, shouldn't its type be `enum`? That's what the team taught me hahahah